.github/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 <h3>New features since last release</h3>
 
+- The number of shots can now be specified on a temporary basis when evaluating a QNode.
+  [(#1075)](https://github.com/PennyLaneAI/pennylane/pull/1075)
```

Comment:
I might also add a short bullet point to breaking changes re: qfuncs that have a shots argument will lead to a deprecation warning :) Just to help us keep track at the next release!